diff --git a/lib/git.zsh b/lib/git.zsh
index 305a77a..bbff153 100644
--- a/lib/git.zsh
+++ b/lib/git.zsh
@@ -3,7 +3,7 @@ function git_prompt_info() {
   if [[ "$(git config --get oh-my-zsh.hide-status)" != "1" ]]; then
     ref=$(command git symbolic-ref HEAD 2> /dev/null) || \
     ref=$(command git rev-parse --short HEAD 2> /dev/null) || return
-    echo "$ZSH_THEME_GIT_PROMPT_PREFIX${ref#refs/heads/}$(parse_git_dirty)$ZSH_THEME_GIT_PROMPT_SUFFIX"
+    echo "$ZSH_THEME_GIT_PROMPT_PREFIX${ref#refs/heads/}$ZSH_THEME_GIT_PROMPT_SUFFIX"
   fi
 }
 
diff --git a/lib/key-bindings.zsh b/lib/key-bindings.zsh
index 49f80c8..1a5424d 100644
--- a/lib/key-bindings.zsh
+++ b/lib/key-bindings.zsh
@@ -19,9 +19,6 @@ bindkey "^[[4~" end-of-line
 bindkey "^[OF" end-of-line
 bindkey ' ' magic-space    # also do history expansion on space
 
-bindkey "^[[1;5C" forward-word
-bindkey "^[[1;5D" backward-word
-
 bindkey '^[[Z' reverse-menu-complete
 
 # Make the delete key (or Fn + Delete on the Mac) work instead of outputting a ~
@@ -41,8 +38,8 @@ bindkey '\C-x\C-e' edit-command-line
 #
 #bindkey '^[[A' up-line-or-search
 #bindkey '^[[B' down-line-or-search
-#bindkey '^[^[[C' emacs-forward-word
-#bindkey '^[^[[D' emacs-backward-word
+# bindkey '^[^[[C' emacs-forward-word
+# bindkey '^[^[[D' emacs-backward-word
 #
 #bindkey -s '^X^Z' '%-^M'
 #bindkey '^[e' expand-cmd-path
diff --git a/themes/pygmalion.zsh-theme b/themes/pygmalion.zsh-theme
index 654e0fc..d35b50c 100644
--- a/themes/pygmalion.zsh-theme
+++ b/themes/pygmalion.zsh-theme
@@ -7,7 +7,7 @@ prompt_setup_pygmalion(){
   ZSH_THEME_GIT_PROMPT_CLEAN=""
 
   base_prompt='%{$fg[magenta]%}%n%{$reset_color%}%{$fg[cyan]%}@%{$reset_color%}%{$fg[yellow]%}%m%{$reset_color%}%{$fg[red]%}:%{$reset_color%}%{$fg[cyan]%}%0~%{$reset_color%}%{$fg[red]%}|%{$reset_color%}'
-  post_prompt='%{$fg[cyan]%}⇒%{$reset_color%}  '
+  post_prompt='%{$fg[cyan]%}->%{$reset_color%}  '
 
   base_prompt_nocolor=$(echo "$base_prompt" | perl -pe "s/%\{[^}]+\}//g")
   post_prompt_nocolor=$(echo "$post_prompt" | perl -pe "s/%\{[^}]+\}//g")
